uiowa/uiowa
# Feature add search api to platform

Resolves #2091 

# To test

 I'm not too worried about the `search_api` module just being turned on affecting functionality negatively, since it is a very commonly used module. Mostly it is necessary to make sure that config is syncing.

* Sync a site, ensure that there are no issues, check basic site functionality.
* @mark-bennett-uiowa Check policy.clas.uiowa.edu to make sure I didn't break any functionality.
* @joewhitsitt Check hr.uiowa.edu to make sure that I didn't break any functionality.
